contributions/catalog/webapps/templates/topbottomnav.html

```diff
@@ -30,4 +30,9 @@ <h1 class="catalog-home">Contributions Catalog</h1>
 
 </div>
 
+<div class="rokwire-footer">
+    <img src= "{{ url_for('static', filename='powered-by-rokwire-no-i-block.png') }}" alt="footer logo" />
```

Comment:
I think we may have to use all images mentioned in the srcset attribute in the design. The current rendering looks slightly blurred. 
<img width="202" alt="image" src="https://user-images.githubusercontent.com/4306604/216075666-43d0778a-60c3-450d-ba52-af8095fc4425.png">

Reply:
updated

Reply:
Thanks.